src/components/views/elements/EditableText.js

```diff
@@ -32,42 +41,65 @@ module.exports = React.createClass({
         Edit: "edit",
     },
 
+    value: '',
+    placeholder: false,
```

Comment:
These are being set on the prototype. I think you meant to do `this.value = '';` and `this.placeholder = false;` in `componentWillMount`. The probably won't be that noticeable since you aren't storing object references like objects or arrays and you clobber correctly via `this.` but it's pretty confusing as it stands.